app/internal/page/check_your_lpa.go

```diff
@@ -70,13 +68,10 @@ func readCheckYourLpaForm(r *http.Request) *checkYourLpaForm {
 func (f *checkYourLpaForm) Validate() validation.List {
 	var errors validation.List
 
-	if !f.Checked {
-		errors.Add("checked", "selectCheckedLpa")
-	}
-
-	if !f.Happy {
-		errors.Add("happy", "selectHappyToShareLpa")
-	}
+	errors.Bool("checked", "checkedLpa", f.Checked,
+		validation.Selected())
+	errors.Bool("happy", "happyToShareLpa", f.Happy,
+		validation.Selected())
```

Comment:
Some of these are kind of only relevant with one type of check, so again, in a future PR I want to tidy up the API  to make more sense